.circleci/scripts/vs_install.ps1

```diff
@@ -1,7 +1,10 @@
-$VS_DOWNLOAD_LINK = "https://aka.ms/vs/15/release/vs_buildtools.exe"
+# https://developercommunity.visualstudio.com/t/install-specific-version-of-vs-component/1142479
+# https://docs.microsoft.com/en-us/visualstudio/releases/2019/history#release-dates-and-build-numbers
+
+# 16.8.5 BuildTools
```

Comment:
According to https://docs.microsoft.com/en-us/visualstudio/releases/2019/history#release-dates-and-build-numbers 16.8 is not among supported baselines. Shouldn't we stick to 16.7 as latest 16.9 is broken?

Reply:
16.8.5 is fairly new and I tested in my local machine.